speechmatics/cli.py

```diff
@@ -218,13 +218,17 @@ def get_transcription_config(
         if args.get(option) is not None:
             config[option] = args[option]
     for option in [
+        "streaming_mode",
         "enable_partials",
         "enable_entities",
         "enable_translation_partials",
         "enable_transcription_partials",
     ]:
         config[option] = True if args.get(option) else config.get(option)
 
+    if args.get("ctrl"):
+        config["ctrl"] = json.loads(args["ctrl"])
```

Comment:
Maybe print out a warning here saying that ctrl is unstable/internal use only

Reply:
Done.